pkg/controller/account/account_controller.go

```diff
@@ -570,14 +570,23 @@ func (r *ReconcileAccount) accountSpecUpdate(reqLogger logr.Logger, account *aws
 
 func (r *ReconcileAccount) nonCCSAssignAccountID(reqLogger logr.Logger, currentAcctInstance *awsv1alpha1.Account, awsSetupClient awsclient.Client) error {
 	// Build Aws Account
-	awsAccountID, err := r.BuildAccount(reqLogger, awsSetupClient, currentAcctInstance)
-	if err != nil {
-		return err
+	var awsAccountID string
+
+	switch utils.DetectDevMode {
+	case utils.DevModeProduction:
+		var err error
+		awsAccountID, err = r.BuildAccount(reqLogger, awsSetupClient, currentAcctInstance)
+		if err != nil {
+			return err
+		}
+	default:
+		log.Info("Running in development mode, skipping account creation")
+		awsAccountID = "123456789012"
```

Comment:
Nice, should shave some time off testing :)

Reply:
This doesn't actually skip anything in our testing as our integration tests have account IDs, so they never actually get to this point, but it allows us to kind of test the accountpool functionality without creating an account. (Each account created by the account pool _will_ fail, but we can test that the account CR is created, etc)